control/pure_pursuit/src/pure_pursuit/pure_pursuit_lateral_controller.cpp

```diff
@@ -185,9 +185,16 @@ double PurePursuitLateralController::calcCurvature(const size_t closest_idx)
 
   if (static_cast<size_t>(closest_idx) >= idx_dist) {
     prev_idx = closest_idx - idx_dist;
+  } else {
+    // return zero curvature because prev_idx can't be found
+    return 0.0;
   }
+
   if (trajectory_resampled_->points.size() - 1 >= closest_idx + idx_dist) {
     next_idx = closest_idx + idx_dist;
+  } else {
+    // return zero curvature because next_idx can't be found
```

Comment:
@kyoichi-sugahara 
IMO
I think it's better to use `prev_idx` and `next_idx` a certain distance far from closest_idx with threshold.
What do you think?
```
 prev_index,<--->closest index <--->next_idx
```

Reply:
@taikitanaka3 
how about this?
>return zero curvature when backward/forward distance is not long enough in the trajectory